coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/core/rest/dates/series/YearPoint.java

```diff
@@ -2,20 +2,17 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonTypeName;
+import lombok.AllArgsConstructor;
 import org.wickedsource.coderadar.core.rest.dates.Year;
 
 @JsonTypeName("year")
+@AllArgsConstructor
```

Comment:
same as with `Day`

Reply:
Unfortunately, this doesn't work as @Value makes the fields final, so we can no longer set them using the setters.